readthedocs/templates/projects/integration_list.html

```diff
@@ -14,6 +14,14 @@
 {% block project_edit_content %}
   <div class="button-bar">
     <ul>
+      {% if connect_vcs_repository %}
+      <li>
+        <form style="float: left" method="post" action="{% url 'projects_integrations_connect_project' project_slug=project.slug %}">
```

Comment:
I think we didn't wan to add inline styles on new code, @agjohnson may have more info about this.